README.md

```diff
@@ -69,10 +69,17 @@ For user space, we need user application debuginfo like `redis-debuginfo-2.8.19-
 For *redhat*\* linux version, we can install as the following:
 
 ```bash
-yum install yum-utils #for debuginfo-install
-yum install systemtap
-yum install kernelname-devel-version
-debuginfo-install kernelname-version
+yum install -y redhat-lsb
```

Comment:
Can `lsb_release` be installed on all *redhat* forked system ? It is just a example to tell user get debuginfo -:)